mobile-app/app/components/TokenIconGroup.tsx

```diff
@@ -20,7 +20,12 @@ interface TokenIconGroupProps {
 
 export function TokenIconGroup (props: TokenIconGroupProps): JSX.Element {
   const additionalIcon = BigNumber.max(props.symbols?.length - props.maxIconToDisplay, 0)
-  const rightOffset = props.offsetContainer === true ? ((props.symbols.length - 1) * -5) - 1 : 0
+  let rightOffset
```

Comment:
can you set this to 0 by default? Then change it if props.offsetContainer is true

Reply:
sure